cluster/examples/kubernetes/ceph/toolbox.yaml

```diff
@@ -28,32 +28,17 @@ spec:
               secretKeyRef:
                 name: rook-ceph-mon
                 key: admin-secret
-        securityContext:
-          privileged: true
         volumeMounts:
-          - mountPath: /dev
-            name: dev
-          - mountPath: /sys/bus
-            name: sysbus
-          - mountPath: /lib/modules
-            name: libmodules
           - name: mon-endpoint-volume
             mountPath: /etc/rook
-      # if hostNetwork: false, the "rbd map" command hangs, see https://github.com/rook/rook/issues/2021
-      hostNetwork: true
+          - name: toolbox
```

Comment:
How about naming the volume `ceph-config`?

Reply:
Sure and that sounds more apt, I'm not good with naming variables :)